base/setup/usetup/muilanguages.h

```diff
@@ -200,8 +200,8 @@
 #ifdef LANGUAGE_ET_EE
     #include "lang/et-EE.h"
 #endif
-// #ifdef LANGUAGE_EU_ES
-    // #include "lang/eu-ES.h"
+#ifdef LANGUAGE_EU_ES
+    #include "lang/eu-ES.h"
 // #endif
```

Comment:
You forgot to uncomment this line.
Please test your translation changes!

Reply:
Sorry, I uploaded a version of the file in which one I haven't uncommented the line. It's a draft yet.